### PR TITLE
ManageSurfaceHeatBal2

### PR DESCRIPTION
**Pull request by mjwitte**
Pull request overview
---------------------
More tinkering with CalcHeatBalanceInsideSurf.

Notes to self:
1. Diffs showed up in 1ZoneEvapCooler, ASHRAE9012016_Hospital_Denver, AirflowNetwork_Multizone_HorizontalOpening, and RefrigeratedWarehouse when TempTerm and TempDev were moved before the iteration loop. Looking at AirflowNetwork_Multizone_HorizontalOpening there's a couple of timesteps early in the first design day that have 280 iterations or so. So the diffs are likely caused by the HConv update every 30 iterations.  But need to investigate why there are soooo many iterations here.

2. Non-adiabatic surfaces with internal source seem to be missing the cross term in TempDiv - need to prove this is wrong before posting a defect.

3. Some (all?) airwalls are showing up in the inside heat balance surface loop - current code just keeps resetting TempIn = TempInOld. These should not be in the list.

4. After reordering, the Zone Component Load Summary report has enormous diffs - setting that aside for now (02 Dec 2019) and focusing on other big diffs (eso and mtr).

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

File: src/EnergyPlus/SolarShading.cc [hunk 1]
```diff
@@ -5791,23 +5774,22 @@ namespace SolarShading {
 
                     if (OverlapArea > 0.001) {
                         ++JBKS;
-                        if (Surface(BackSurfNum).BaseSurf == BackSurfNum) JBKSbase = JBKS;
```
Comment (mjwitte) [inline src/EnergyPlus/SolarShading.cc:5794]: @amirroth Found one sneaky Base Surface reference that relied on sub-surfaces following directly after. Fixes big diffs in Exercise1B-Solution (1 zone, 6 surfaces, nice one to debug). But does not fix the Zone Component Loads Report in 5ZoneAirCooled_ZoneAirMassFlowBalance.

**Comment by nrel-bot-3** [comment]
@mjwitte @lgentile it has been 28 days since this pull request was last updated.

**Comment by JasonGlazer** [comment]
@mjwitte did you mean to close this pull request?

**Comment by mjwitte** [comment]
Temporarily, yes. For some reason, CI is repeatedly running tests on external branches, even if nothing changes. So, this is closed until I'm ready to work on it again.

**Comment by mjwitte** [comment]
Closing this for good. Replaced with #7847.

**Pull request closed without merging**